NEWS.txt

```diff
@@ -14,7 +16,6 @@ Version 7.41 - 2023/12/21
 Version 7.40 - 2023/11/02
 * user interface
   - Added infobox that combines ETA with AAT dT
-  - FLARM: Add 100m zoom option.
```

Comment:
????

Reply:
Please do not remove that line. Please rebase to current head